RussianFilter/sections/specific.txt

```diff
@@ -23,6 +23,7 @@ livejournal.com##.b-singlepost-afterpostbanner
 livejournal.com##.lj-card--banner-context-first
 livejournal.com##.pagewide-wrapper
 gmt-max.net##div[class^="block_teaser_"]
+ntsk.ru##.right > div.right-cell:not(.services)
```

Comment:
better use `div[class="right-cell"]:not(.services)` instead of `div.right-cell:not(.services)` in this case